agent/outbound_test.go

```diff
@@ -138,10 +138,11 @@ func Test_addAppDeletionToQueue(t *testing.T) {
 		defer func() {
 			a.mode = types.AgentModeAutonomous
 		}()
-		// App must be already managed for event to be generated
+		// Delete events will be sent for managed applications on managed agent, whether or not the application is managed.
```

Comment:
The previous comment on this test was incorrect: 
- The current product code allow apps events to be generated for non-managed app (see the function under test)
- The test claimed to prove that unmanaged applications did NOT send events, but actually it was  passing for an unrelated reason.